### DISCUSSION TITLE
[Projects Beta] Attach a `README` to a project

### DISCUSSION THREAD

**Discussion by MarkBennett** [community/community#10131] [Projects and Issues] [answered]
To make documenting workflow and practices easier, a description that you could attach to a project would let you orient and on-board new members and could contain links to discussions about workflow and issues. This could be in a `README` tab at the top of the project board and should support GitHub flavoured Markdown.

**Comment by coot** [comment]
This would also make it easier to list all views.  For larger projects, the horizontal list on a project page is not that easy to use when there are dozens of views.

**Comment by azenMatt** [comment, marked as answer]
Hi @MarkBennett - we just shipped a new description and readme field - https://github.blog/changelog/2022-02-02-the-new-github-issues-02-02-update/ 

Let me know if this helps!

**Reply by MarkBennett** [reply]
Those look great @azenMatt! Thanks for sharing the update with me as I'd not seen it earlier this week! Great working shipping! 🙂
